### PR TITLE
just fixing indenting from Kyra's last comment no actual code changes 

### PR DESCRIPTION
**Pull request by hengelman**

### CODE REVIEW

**Review by kyrabusser — APPROVED** [review]
✨ ✨ ✨

File: src/main/webapp/feed.html [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+	<link rel="stylesheet" href="/css/main.css">
+	<link rel="stylesheet" href="/css/user-page.css">
+	<script src="/js/feed-loader.js"></script>
+  </head>
+  <body onload="buildUI()">
+ 	<div id="content">
+ 	  <h1>Message Feed</h1>
+          <hr/>
+  	<div id="message-container">Loading...</div>
+    </div>
```
Comment (kyrabusser) [inline src/main/webapp/feed.html:14]: Ooh, actually, this should line up with the `<div id="content">` it's closing from line 10. Anyway, thanks for cleaning this up! 👍

Comment (hengelman) [reply src/main/webapp/feed.html:14]: screen shot 2019-03-02 at 4 25 12 pm
Yeah I'm confused bc on my text editor everything lines up but then when I pushed it everything is messed up again? I don't know why sublime doesn't like HTML

Comment (kyrabusser) [reply src/main/webapp/feed.html:14]: Hmm--maybe a tab vs spaces thing? It's tricky to see in GitHub, but from trying to select the whitespace that's one thing I'd check. Maybe try setting `translateTabsToSpaces` true? http://www.sublimetext.com/docs/indentation

**Pull request merged**